### PR TITLE
Add animation reference to filter in Viz docs

### PR DESCRIPTION
**Pull request by davidmanzanares**
Fix https://github.com/CartoDB/carto-vl/issues/606

### CODE REVIEW

File: src/Viz.js [hunk 1]
```diff
@@ -42,6 +42,24 @@ const SUPPORTED_PROPERTIES = [
     'variables'
 ];
 
+/**
+ * A vizSpec object is used to create a {@link carto.Viz|Viz} and controlling multiple aspects.
+ * For a better understanding we recommend reading the {@link TODO|VIZ guide}
```
Comment (Jesus89) [inline src/Viz.js]: Could we add now a link for this?

Comment (davidmanzanares) [reply src/Viz.js]: We should, although I don't know how should we point to a guide. @csubira do you know how should we do it?

Comment (csubira) [reply src/Viz.js]: I'd suggest following the same pattern that here in carto-js https://github.com/CartoDB/carto.js/blob/master/src/api/v4/filter/bounding-box-gmaps.js#L11 with the full link. Would that work for you?

Comment (davidmanzanares) [reply src/Viz.js]: :+1:

**Pull request merged**